### PR TITLE
chore: implement lightweight test listing

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-test/src/runner.ts [hunk 1]
```diff
@@ -163,10 +163,27 @@ export class Runner {
...
+    const report: any = {
+      projects: []
+    };
+    for (const [project, files] of filesByProject) {
+      report.projects.push({
+        name: project.config.name,
+        files: files
+      });
+    }
+    process.stdout.write(JSON.stringify(report));
```
Comment (dgozman) [inline packages/playwright-test/src/runner.ts:177]: I'd rather move output to the cli.

File: packages/playwright-test/src/cli.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ export function addTestCommand(program: Command) {
   command.option('--global-timeout <timeout>', `Maximum time this test suite can run in milliseconds (default: unlimited)`);
   command.option('-j, --workers <workers>', `Number of concurrent workers, use 1 to run in a single worker (default: number of CPU cores / 2)`);
   command.option('--list', `Collect all the tests and report them, but do not run`);
+  command.option('--list-files', `High level list`);
```
Comment (dgozman) [inline packages/playwright-test/src/cli.ts:47]: Why both this argument, and a separate command?

File: packages/playwright-test/src/cli.ts [hunk 3]
```diff
@@ -73,6 +74,21 @@ Examples:
   $ npx playwright test --browser=webkit`);
 }
 
+export function addListTestsCommand(program: Command) {
+  const command = program.command('list-tests [test-filter...]', { hidden: true });
+  command.description('Run tests with Playwright Test');
```
Comment (dgozman) [inline packages/playwright-test/src/cli.ts:79]:
```suggestion
  command.description('List test files');
```

File: packages/playwright-test/src/cli.ts [hunk 4]
```diff
@@ -73,6 +74,21 @@ Examples:
...
+  command.option('-c, --config <file>', `Configuration file, or a test directory with optional ${kDefaultConfigFiles.map(file => `"${file}"`).join('/')}`);
+  command.option('--project <project-name...>', `Only run tests from the specified list of projects (default: run all projects)`);
```
Comment (dgozman) [inline packages/playwright-test/src/cli.ts:81]:
```suggestion
  command.option('--project <project-name...>', `Only list test files from the specified list of projects (default: all projects)`);
```

File: packages/playwright-test/src/cli.ts (hunk 2 above)
Comment (pavelfeldman) [reply packages/playwright-test/src/cli.ts:47]: leftover, thanks.

**Pull request merged**